docusaurus.config.js

```diff
@@ -107,11 +107,11 @@ const config = {
             items: [
               {
                 label: 'Discord',
-                href: 'https://discord.gg/n6ebKsX46w',
+                to: 'https://discord.gg/obol',
               },
               {
                 label: 'Twitter',
-                href: 'https://twitter.com/obolnetwork',
+                to: 'https://twitter.com/obolnetwork',
```

Comment:
Why is href changing to `to` here? Also same change on the discord link

Reply:
basically removes the arrow in the footer, as some links had it while others didn't